pkgs/development/libraries/mesa/default.nix

```diff
@@ -51,9 +51,9 @@
       "imagination-experimental" # PowerVR Rogue (currently N/A)
       "panfrost" # ARM Mali Midgard and up (T/G series)
     ]
-    ++ lib.optionals stdenv.isx86_64 [
-      "intel" # Intel (aka ANV), could work on non-x86_64 with PCIe cards, but doesn't build as of 22.3.4
-      "intel_hasvk" # Intel Haswell/Broadwell, experimental, x86_64 only
+    ++ lib.optionals (stdenv.isx86_64 || stdenv.isx86_32) [
```

Comment:
Nit: perhaps just `lib.optionals stdenv.hostPlatform.isx86` as above?

Reply:
That one does not exists:

```
nix-repl> pkgs.stdenv.is    
pkgs.stdenv.is32bit      pkgs.stdenv.isAarch64    pkgs.stdenv.isCygwin     pkgs.stdenv.isLinux      pkgs.stdenv.isSunOS      pkgs.stdenv.isx86_64
pkgs.stdenv.is64bit      pkgs.stdenv.isBSD        pkgs.stdenv.isDarwin     pkgs.stdenv.isMips       pkgs.stdenv.isi686
pkgs.stdenv.isAarch32    pkgs.stdenv.isBigEndian  pkgs.stdenv.isFreeBSD    pkgs.stdenv.isOpenBSD    pkgs.stdenv.isx86_32
```

Reply:
`stdenv.hostPlatform.isx86` is what I ended up with for the OpenGL bits

Reply:
I don't know why there are these inconsistencies between `stdenv.*` and `stdenv.hostPlatform.*`.

Reply:
Got confused by that...